arctic/chunkstore/chunkstore.py

```diff
@@ -360,16 +359,17 @@ def write(self, symbol, item, metadata=None, chunker=DateChunker(), audit=None,
                             START: start,
```

Comment:
line 357 (`op = True`) can be deleted
minor: eliminate `find` variable for consistency